tensorflow/core/kernels/segment_reduction_ops_impl.h

```diff
@@ -266,10 +266,10 @@ class SegmentSumGPUOp : public AsyncOpKernel {
                          sizeof(Index))
             .ok(),
         errors::Internal(
-            "SegmentSumGPUOp: failed to copy output_rows from device"),
+            "SegmentReductionGPUOp: failed to copy output_rows from device"),
```

Comment:
This should use the name of the operation, not `SegmentReductionGPUOp`.  you can use `type_string`.

Reply:
Thanks for your advice! Have done.